src/pages/ThorChainLP/Pool/Pool.tsx

```diff
@@ -94,70 +87,61 @@ export const Pool = () => {
   const translate = useTranslate()
   const history = useHistory()
 
-  const { data: parsedPools } = usePools()
-
-  const foundPool = useMemo(() => {
-    if (!parsedPools) return undefined
-    const routeOpportunityId = decodeURIComponent(params.poolOpportunityId ?? '')
+  const { data: pools } = usePools()
 
-    return parsedPools.find(pool => pool.opportunityId === routeOpportunityId)
-  }, [params, parsedPools])
+  const pool = useMemo(() => {
+    return pools?.find(pool => pool.asset === params.poolAssetId)
+  }, [params.poolAssetId, pools])
 
   const { isTradingActive, isLoading: isTradingActiveLoading } = useIsTradingActive({
-    assetId: foundPool?.assetId,
-    enabled: !!foundPool,
+    assetId: pool?.assetId,
+    enabled: !!pool,
     swapperName: SwapperName.Thorchain,
   })
 
   const poolAssetIds = useMemo(() => {
-    if (!foundPool) return []
-
-    return [foundPool.assetId, thorchainAssetId]
-  }, [foundPool])
+    if (!pool) return []
 
-  const headerComponent = useMemo(
-    () => <PoolHeader assetIds={poolAssetIds} name={foundPool?.name ?? ''} />,
-    [foundPool?.name, poolAssetIds],
-  )
+    return [pool.assetId, thorchainAssetId]
+  }, [pool])
 
   const handleAddLiquidityClick = useCallback(() => {
     history.push(
-      generatePath('/pools/add/:poolOpportunityId', {
-        poolOpportunityId: foundPool?.opportunityId ?? '',
+      generatePath('/pools/add/:poolAssetId', {
```

Comment:
Note, from a UX perspective this means we will route to the default opportunity for adding liquidity, not the specific a/sym side of said pool